cfme/tests/control/test_alerts.py

```diff
@@ -216,8 +214,8 @@ def test_alert_vm_turned_on_more_than_twice_in_past_15_minutes(
     provider.refresh_provider_relationships()
 
     # preparing events to listen to
-    builder = EventBuilder(get_or_create_current_appliance())
-    base_evt = partial(builder.new_event, target_type='VmOrTemplate', target_name=vm_name)
+    listener = appliance.event_listener()
+    base_evt = partial(listener.new_event, target_type='VmOrTemplate', target_name=vm_name)
```

Comment:
register_event currently accepts any number of event objects and works on top of listener.listen_to() method. 
Event objects has to be created somehow in order to be passed to register_event.
The listener.new_event() method prepares such event objects.

Actually,  I can combine new_event + register_event if you want. The minor drawback is register_event will accept only one event (list of event parameters) and  it will be a bit less obvious.

Frankly speaking, I don't like register_event fixture at all. its aim isn't clear. I think this should be completely replaced. We have already discussed replacement recently.